jsparagus/emit/rust.py

```diff
@@ -529,19 +541,18 @@ def map_with_offset(args, unpack):
                     assert len(act.args) == 0
                     self.write(indent, "return Ok(true);")
                     return False
-                elif has_ast_builder:
-                    # TODO: Check whether the function is implemented by the
-                    # given traits, to decide whether to implement it or not.
+                else:
+                    delegate = ""
+                    if str(act.trait) == "AstBuilder":
```

Comment:
would be nice to add global constant for this, if this is special value instead of just type name,
so it's clear what `"AstBuilder"` actually means in the code that specifies it.

Reply:
Ideally it is not meant to be a special value. However, this is a legacy caused by the fact that we already have a lot of code in the `ASTBuilder` file, and that we have the real trait is `ASTBuilderDelegate` to avoid some large changes, which are time consuming.